protos/index.proto

```diff
@@ -163,8 +155,6 @@ message VectorIndexStage {
     Transform transform = 4;
     // DiskANN
     DiskAnn diskann = 5;
-    // HNSW
```

Comment:
Where are these stored ?

Reply:
they are gone, the new index file is in lance format, we can store all params (e.g. index type, index name, index's params) in the lance's metadata.
this only works for old index format, HNSW will be on a new way

Reply:
make it more clear:
the metadata of the index file of `IVF_HNSW`, contains the metadata of IVF by key `IVF_METADATA_KEY`, and contains the partitions' metadata in a list by key `IVF_PARTITION_KEY`
the PQ will be stored in aux file, its metadata contains the PQ metadata.